### ISSUE TITLE
`@graphql-codegen/cli` shouldn't have dependency on a specific `graphql` version

### ISSUE THREAD

**Issue by vhenzl** [dotansimha/graphql-code-generator#2031]
PR #2003 added `graphql` version 14.3.1 as dependency to `@graphql-codegen/cli`. I believe that's a mistake as `graphql` is already specified in `peerDependencies`.

In my project, I have some custom plugins and installed `graphql@14.3.0`. Running `graphql-codegen` after upgrade to v1.3.0 fails with errors like this:

```
    Error: Cannot use GraphQLObjectType "Query" from another module or realm.

    Ensure that there is only one instance of "graphql" in the node_modules
    directory. If different versions of "graphql" are the dependencies of other
    relied on modules, use "resolutions" to ensure only one version is installed.

    https://yarnpkg.com/en/docs/selective-version-resolutions

    Duplicate "graphql" modules cannot be used at the same time since different
    versions may have different capabilities and behavior. The data from one
    version used in the function from another could produce confusing and
    spurious results.

```

The reason is, that plugins use `graphql@14.3.0` from the project but `graphql-codegen` loads `graphql@14.3.1` because of the explicit dependency on the specific version.

Updating project's dependency to `graphql@14.3.1` fixes the problem, but IMO `@graphql-codegen/cli` shouldn't require a specific version of `graphql`.

**Comment by dotansimha** [comment]
You are right @vhenzl ! 
@kamilkisiela can you please take a look?

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/commit/1c4d88c90a13dc5fd9befdd1185a85180dbd09d2

**Comment by dotansimha** [comment]
Fixed in 1.3.1
